python/ray/actor.py

```diff
@@ -659,29 +585,7 @@ def _get_or_create_impl(self, args, kwargs):
         return ActorOptionWrapper()
 
     @_tracing_actor_creation
-    def _remote(
-        self,
-        args=None,
-        kwargs=None,
-        num_cpus=None,
-        num_gpus=None,
-        memory=None,
-        object_store_memory=None,
-        resources=None,
-        accelerator_type=None,
-        max_concurrency=None,
-        max_restarts=None,
-        max_task_retries=None,
-        name=None,
-        namespace=None,
-        lifetime=None,
-        placement_group="default",
-        placement_group_bundle_index=-1,
-        placement_group_capture_child_tasks=None,
-        runtime_env=None,
-        max_pending_calls=-1,
-        scheduling_strategy: SchedulingStrategyT = None,
-    ):
+    def _remote(self, args=None, kwargs=None, **actor_options):
```

Comment:
The `actor_options` here is already merged options? If so we probably don't need the extra merge that's happening in this method (maybe you already planned it as a follow-up).

Reply:
are you referring to

```python
        # fill actor required options
        for k, v in options.actor_options.items():
            actor_options[k] = actor_options.get(k, v.get_default_value())
```
?

here we are not merging it again, just fulfilling missing options. we did similar stuff to `__ray_metadata__`, but unfortunately (and annoyingly) `__ray_metadata__` lacks a few fields, so we still need to fill missing options again. I decide to clean it up later.

Reply:
We have code like `scheduling_strategy = scheduling_strategy or meta.scheduling_strategy` which I think is no longer needed?

Reply:
yes, that's true. let me clean these fields